### DISCUSSION TITLE
Keep getting this error when I try to install Homebrew using the terminal. Can anyone help?

### DISCUSSION THREAD

**Discussion by k0Dwo** [Homebrew/discussions#3263] [Getting started]
error: RPC failed; curl 56 LibreSSL SSL_read: error:02FFF036:system library:func(4095):Connection reset by peer, errno 54
error: 1394 bytes of body are still expected
fetch-pack: unexpected disconnect while reading sideband packet
fatal: early EOF
fatal: fetch-pack: invalid index-pack output
Failed during: git fetch --force origin
<img width="1258" alt="Screenshot 2022-05-12 at 22 35 55" src="https://user-images.githubusercontent.com/77553554/168178965-a47e9e84-7412-457c-9615-562663f325f9.png">

**Comment by SMillerDev** [comment]
See #666

**Comment by ask10000** [comment]
Seeing this issue on a new mac with M1 (Monterey v 12.4). Older mac with same OS version doesn't seem to have this problem. Both the devices are on same network - no VPN. So this is clearly not a network issue as well.
